configuration/blockbook/blockchaincfg.json

```
@@ -21,5 +21,6 @@
 
     "mempool_workers": 8,
     "mempool_sub_workers": 2,
+    "mempool_filter_scripts": "taproot",
```

I think `golomb_filter_p` param is missing. Without it, no filters are ever created.
```suggestion
    "mempool_filter_scripts": "taproot",
    "golomb_filter_p": 20,
```

---

fixed forcepushed